.github/workflows/hil_tests.yml

```
@@ -110,22 +110,26 @@ jobs:
           - hil_board: mimxrt1024_evk
             west_board: mimxrt1024_evk
             manifest: .ci-west-zephyr.yml
+            extra_twister_args: '--west-flash="--dev-id=${!SNR_VAR}"'
             run_tests: true
 
           - hil_board: nrf52840dk
             west_board: nrf52840dk_nrf52840
             manifest: .ci-west-zephyr.yml
+            extra_twister_args: '--west-flash="--dev-id=${!SNR_VAR}"'
             run_tests: true
 
           - hil_board: nrf9160dk
             west_board: nrf9160dk_nrf9160_ns
             manifest: .ci-west-ncs.yml
+            extra_twister_args: '--west-flash="--dev-id=${!SNR_VAR}"'
             run_tests: true
 
           - hil_board: rak5010_nrf52840
```

```suggestion
          - hil_board: rak5010
```

Use shorted board name. This matches the self-hosted runner board recognition script